drivers/filesystems/fs_rec/fs_rec.c

```diff
@@ -468,6 +474,17 @@ DriverEntry(IN PDRIVER_OBJECT DriverObject,
                              0);
     if (NT_SUCCESS(Status)) DeviceCount++;
 
+    /* Register FATX */
+    Status = FsRecRegisterFs(DriverObject,
+                             NULL,
+                             NULL,
+                             L"\\FatX",
```

Comment:
Regarding this, in the (now old) fastfat, I think this name should be changed (to "FatX") as well:
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/filesystems/fastfat/iface.c;hb=e5ebbc8afaf104eff2db274a73131e83e6bed4dc#l55
(and we can get rid of the "RosFat" case in that function as well.)